build.sbt

```diff
@@ -10,8 +10,7 @@ def previousVersion = "0.7.0"
 def scala213 = "2.13.4"
 def scala212 = "2.12.13"
 def scala211 = "2.11.12"
-def scala3Stable = "3.0.0-RC3"
```

Comment:
Let's leave RC3 for a while if possible.

Reply:
One thing we could do is to drop the support, but backpublish 3.0.0 for 0.7.25. So we the next release would drop RC3, but in the meantime we can keep using 0.7.25 for both 3.0.0 and 3.0.0-RC3

Reply:
We can work around it in Metals for sure, just last time we dropped to one version there was a bit of a backlash, since people would need to update munit and Scala version at the same time.

Reply:
Last time I think @djspiewak had some opinion about this. Maybe also @larsh can chime in?

Reply:
I think it makes sense to support the old and new version for at least one release. It simplifies upgrading in many cases.

Reply:
@olafurpg wdyt about backpublishing 3.0.0 for the current version and merging the changes as is? This way we can move forward in a clean way (scalacheck recently dropped support for 2.11 in a similar fashion)

Reply:
That sounds fine. I generally avoid backpublishing since it requires manual local steps that’s easy to mess up compared to automated CI releases. It’s a bummer because backpublishing is very user friendly.

Reply:
1. Yes, please keep two versions, it makes upgrading easier.
2. I try to backpublish wherever I can (at least ScalaCheck and Discipline). Given the choice between "harder for me" and "harder for users", I pick "harder for me" :wink:

Reply:
I'll try backpublishing. I've done it quite a few times in the past week for some internal projects, so I'm more confident than I usually would be in doing so :)

Reply:
If that works, I could backpublish all the way to Cats <3

Reply:
so, I've made a mistake backpublishing (feeling too confident, I knew it!) plus it turns out that backpublishing other typelevel libraries would be hard (due to old sbt-dotty versions), so I've restored RC3 support and I'll merge this and cut 0.7.26